fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -24,6 +25,13 @@
 
 #ifdef SSD_ROCKSDB_EXPERIMENTAL
 
+static_assert(ROCKSDB_MAJOR >= 6
```

Comment:
does this enforce rocksDB version must be 6.22.1?
If we enforce the exact version,  can we report error if "ROCKSDB_MAJOR. ROCKSDB_MINOR. ROCKSDB_PATCH != 6.22.1" and print out ROCKSDB_MAJOR. ROCKSDB_MINOR. ROCKSDB_PATCH value?

Reply:
@xumengpanda, In this file I added a static assert to ensure the version is greater than or equal to 6.22.1